app/_data/plugins.yml

```diff
@@ -86,6 +96,10 @@
   info: Transform request and responses on the fly on Kong
   plugins:
     - name: Request Transformer
+      info: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
info: Use powerful regular expressions, variables and templates to transform API requests.